std/range/primitives.d

```diff
@@ -2055,9 +2055,13 @@ if (isBidirectionalRange!Range)
 }
 
 /**
-   Moves the front of `r` out and returns it. Leaves `r.front` in a
-   destroyable state that does not allocate any resources (usually equal
-   to its `.init` value).
+   Moves the front of `r` out and returns it.
+
+   If `r.front` is a struct with a destructor or postblit defined, it is reset
```

Comment:
This is slightly confusing as is because it uses "it" for both `r.front` (as it lives in the range `r`) and the returned/moved value.

Reply:
I think that we should mention the copy constructor here instead of the postblit.

Reply:
> I think that we should mention the copy constructor here instead of the postblit.

Yes, the documentation for `core.lifetime.move` is itself outdated too in that regard.

Reply:
Changed to "copy constructor" and tried to clarify the wording a bit.